sql/comment-syntax.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Comment Syntax
+category: user guide
+---
+
+# Comment Syntax
+
+TiDB supports three comment styles:
+
+- Use `#` to comment a line.
+- Use `--` to comment a line, and this style requires at least one whitespace before `--`.
+- Use `/* */` to comment a block and multiple lines.
+
+Example:
+
+```
+mysql> SELECT 1+1;     # This comment continues to the end of line
++------+
+| 1+1  |
++------+
+|    2 |
++------+
+1 row in set (0.00 sec)
+
+mysql> SELECT 1+1;     -- This comment continues to the end of line
++------+
+| 1+1  |
++------+
+|    2 |
++------+
+1 row in set (0.00 sec)
+
+mysql> SELECT 1 /* this is an in-line comment */ + 1;
++--------+
+| 1  + 1 |
++--------+
+|      2 |
++--------+
+1 row in set (0.01 sec)
+
+mysql> SELECT 1+
+    -> /*
+   /*> this is a
+   /*> multiple-line comment
+   /*> */
+    -> 1;
++-------+
+| 1+
+
+1 |
++-------+
+|     2 |
++-------+
+1 row in set (0.00 sec)
+
+mysql> SELECT 1+1--1;
++--------+
+| 1+1--1 |
++--------+
+|      3 |
++--------+
+1 row in set (0.01 sec)
+```
```

Comment:
The Chinese text has been updated. Please see https://github.com/pingcap/docs-cn/pull/347